src/helper/ajax.js

```diff
@@ -8,16 +8,17 @@ const cookies = new Cookies();
 const obj = {};
 
 ['get', 'post', 'postJSON', 'all'].forEach(function(method) {
-  obj[method] = function(url, payload, settings = {}) {
+  obj[method] = function(url, payload, settings) {
     /* Request will be aborted after 30 seconds */
     settings = {
       timeout: 30000,
       dataType: 'json',
       crossDomain: true,
+      externalAPI: false,
```

Comment:
Rename it to `isTokenRequired`. It would even work for the cases where token doesn't needs to be passed in SUSI.AI APIs.